pallets/identity/src/lib.rs

```diff
@@ -1453,8 +1453,7 @@ impl<T: Config> Module<T> {
         };
         <DidRecords<T>>::insert(&did, record);
 
-        // 2.3. Give 100k POLYX to the primary key for testing.
-        // TODO: Remove before mainnet.
+        // 2.3. Give `InitialPOLYX` to the primary key for testing.
         T::Balances::deposit_creating(&sender, T::InitialPOLYX::get().into());
```

Comment:
`deposit_creating` does a zero check so doing it here may not make sense, as it is just an additional check.